src/views/SeedPhrase/ConfirmSeed.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+
+    <div class="login-wrapper confirm-seed h-100 no-outer-pad">
+        <div class="confirm-seed_logo-wrap mt-5 mb-3 mx-auto">
+            <img :src="logo"/>
+        </div>
+        <div class="confirm-seed_top">
+            <h2>Confirm Seed Phrase</h2>
+            <p class="px-3">Tap the 3 words matching their position in the seed phrase.  Once confirmed, store the phrase securely.</p>
+        </div>
+        <div class="confirm-seed_bottom">
+            <div class="confirm-seed_bottom-top">
+                <div class="mx-auto mt-2">
+                <form class="form">
+                    <div class="confirm-seed_inputs d-flex flex-row justify-content-around">
+                        <div>
+                            <label>1<sup>st</sup> Word</label>
```

Comment:
I believe these should be randomised so users see a different index to confirm each time